Alignment/OfflineValidation/src/Trend.cc

```diff
@@ -210,29 +203,26 @@ Trend::Trend(const char* name,
       auto currentRun = run.second.get_value<int>();
 
       auto lumi = GetLumi(GetLumi.firstRun, currentRun);
-      //cout << currentRun << '\t' << lumi << endl;
+
       if (lumi > 0)
         v->DrawLine(lumi, ymin, lumi, ymax);
     }
   }
 }
 
 void Trend::operator()(TObject* obj, TString drawOpt, TString lgdOpt, bool fullRange) {
-  cout << __func__ << endl;
   c.cd();
 
   TString classname = obj->ClassName();
   if (classname.Contains("TGraph")) {
     auto g = dynamic_cast<TGraph*>(obj);
     int n = g->GetN();
-    cout << g->GetPointX(n - 1) << ' ' << GetLumi.lastRun;
+
     if (fullRange) {
-      cout << " -> adding one point" << endl;
       g->Set(n);
       g->SetPoint(n, GetLumi.lastRun, 0);
     } else
-      cout << " -> hole between end of graph and right edge" << endl;
-    g = GetLumi(g);
+      g = GetLumi(g);
```

Comment:
This is now depending on the `else` in the previous line, which is not what was there before. and quite likely not what you want

Reply:
Right, good catch, thank you! Will update it